src/components/Header/Navbar.js

```diff
@@ -0,0 +1,54 @@
+/* eslint-disable react/button-has-type */
+// @ts-nocheck
+import React, { useContext } from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faGear, faMicrophone } from '@fortawesome/free-solid-svg-icons';
+import Nav from 'react-bootstrap/Nav';
+import AuthContext from '../../redux/auth-context';
+
+import classes from './Navbar.module.css';
+
+const Navbar = () => {
+  const ctx = useContext(AuthContext);
+  return (
+    <>
+      {ctx.isLoggedIn && (
+        <Nav defaultActiveKey="/home" className={classes.nav} as="ul">
+          <Nav.Item as="li">
+            <span>Search</span>
+          </Nav.Item>
+          <Nav.Item as="li">
```

Comment:
- [ ] **Optional:** Kindly implement a filter here. This will enable the search field and display the data on the page.